packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -68,13 +68,20 @@ const defaultProps = {
   selectedDate: undefined,
 };
 
+const contextTypes = {
+  /* eslint-disable consistent-return */
+  intl: (context) => {
```

Comment:
I think using a required intl prop would be less code but the reason for using this pattern to check for it is explained by this comment: https://github.com/cerner/terra-core/pull/431/files#r120640131